ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/OntologiesViewImpl.java

```diff
@@ -111,6 +111,14 @@
     private OntologyUtil ontologyUtil = OntologyUtil.getInstance();
     private PagingLoader<FilterPagingLoadConfig, PagingLoadResult<App>> loader;
 
+    boolean ontologySelected;
+    boolean isActiveOntology;
+    boolean editorTreeHasSelection;
+    boolean hasAppSelected;
+    boolean isExternalApp;
+    boolean isDeletedApp;
+    boolean editorTreeHasHierarchies;
```

Comment:
I don't think we need to keep track of this state outside of `updateButtonStatus`, so these can just be local vars in that method.